modules/installation-launching-installer-custom.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_aws/installing-aws-network-customizations.adoc
+// * installing/installing_azure/installing-azure-customizations.adoc
+// If you use this module in any other assembly, you must update the ifeval
+// statements.
+
+[id="installation-launching-installer-custom_{context}"]
+= Deploy the cluster
+
+You can install {product-title} on a compatible cloud.
+
+[IMPORTANT]
+====
+You can run the installation program only once, during initial installation.
+====
+
+.Prerequisites
+
+* Configure an account with the cloud provider to host your cluster.
+* Obtain the {product-title} installation program and the pull secret for your
+cluster.
+
+.Procedure
+
+. Run the installation program:
++
+----
+$ ./openshift-install create cluster --dir=<installation_directory> \ <1>
```

Comment:
why do we tell people to `--log-level debug` we should let them see what's expected with default which is info.

Reply:
It looks like the change came in after the review of the architecture content.

Do you want line 31 to read
```
    --log-level info <2>
```
or something else?

Reply:
I'm assuming @abhinavdahiya means to just leave it off. When log level isn't specified, the default is info.

Reply:
I'm opening a separate PR to remove this option: https://github.com/openshift/openshift-docs/pull/16556

This will make it easier for me to apply the change to 4.1 as well.